Core/Node-API/Include/Shared/napi/js_native_api.h

```diff
@@ -528,8 +529,40 @@ node_api_post_finalizer(napi_env env,
                         void* finalize_data,
                         void* finalize_hint);
 
+NAPI_EXTERN napi_status napi_create_bigint_int64(napi_env env,
+                                                 int64_t value,
+                                                 napi_value* result);
+NAPI_EXTERN napi_status napi_create_bigint_uint64(napi_env env,
+                                                  uint64_t value,
+                                                  napi_value* result);
+NAPI_EXTERN napi_status napi_create_bigint_words(napi_env env,
+                                                 int sign_bit,
+                                                 size_t word_count,
+                                                 const uint64_t* words,
+                                                 napi_value* result);
+NAPI_EXTERN napi_status napi_get_value_bigint_int64(napi_env env,
+                                                    napi_value value,
+                                                    int64_t* result,
+                                                    bool* lossless);
+NAPI_EXTERN napi_status napi_get_value_bigint_uint64(napi_env env,
+                                                     napi_value value,
+                                                     uint64_t* result,
+                                                     bool* lossless);
+NAPI_EXTERN napi_status napi_get_value_bigint_words(napi_env env,
+                                                    napi_value value,
+                                                    int* sign_bit,
+                                                    size_t* word_count,
+                                                    uint64_t* words);
 #endif  // NAPI_EXPERIMENTAL
 
+// [BABYLON NATIVE]: Promote this function from experimental as a stopgap
+NAPI_EXTERN napi_status napi_add_finalizer(napi_env env,
+                                           napi_value js_object,
+                                           void* native_object,
+                                           napi_finalize finalize_cb,
+                                           void* finalize_hint,
+                                           napi_ref* result);
+
```

Comment:
There is already another version in this file. We need to consolidate it somehow.

Reply:
I'm removing the definition with the `[BABYLON NATIVE]` tag as the function is part of napi update as well.

Reply:
Yes, but it's under a version define. We are currently only defining V3.

Reply:
It builds and unittests are ok with `NAPI_VERSION=5`  and  `NAPI_HAS_THREADS=0`